Source/Dafny/Resolver.cs

```diff
@@ -16928,6 +16926,10 @@ class DefaultValueSubstituter : Substituter {
       return ok && ctor.Formals.Count == dtv.Arguments.Count;
     }
 
+    public static string IsGhostPrefix(bool isGhost) {
```

Comment:
Would a better name for this method be `GhostPrefix`?

Reply:
I agree. I renamed it.